packages/dma-contracts/scripts/deployment/deploy.ts

```diff
@@ -362,11 +362,14 @@ export class DeploymentSystem extends DeployedSystemHelpers {
     let configString = inspect(this.config, { depth: null })
     configString = this.replaceServiceRegistryName(configString, this.findStringPath)
 
+    const networkEnumString =
+      this.network !== Network.TENDERLY
+        ? this.getNetworkEnumString(this.network)
+        : this.getNetworkEnumString(Network.MAINNET)
```

Comment:
I tihnk that's easier to read? ;p
```suggestion
      this.network === Network.TENDERLY
        ? this.getNetworkEnumString(Network.MAINNET)
        : this.getNetworkEnumString(this.network)
```

Reply:
Agree!